sympy/polys/matrices/domainmatrix.py

```diff
@@ -76,6 +173,21 @@ def unify(self, other):
         return self, other
 
     def to_Matrix(self):
+        """ Converts an object of Domain Matrix class to
+        sympy's Matrix Class
+
+        Examples
+        ========
+
+        >>> from sympy import ZZ
+        >>> from sympy.polys.matrices import DomainMatrix
+        >>> A = DomainMatrix([[1, 2], [2, 3]], (2, 2), ZZ)
```

Comment:
This should be
```
A = DomainMatrix([[ZZ(1), ZZ(2)], [ZZ(2), ZZ(3)]], (2, 2), ZZ)
```
This applies to all examples that create a DomainMatrix directly. It must only be created with domain elements.